sktree/datasets/hyppo.py

```diff
@@ -236,6 +238,7 @@ def make_trunk_mixture_classification(
     band_type: str = "ma",
     return_params: bool = False,
     mix: float = 0.5,
+    scaling_factor: Optional[float] = None,
```

Comment:
Thoughts on putting this for all 3 simulation functions? The default scaling factor for the marron and wand can be whatever that simulations' factor currently is